spec/active_record/column_methods_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+
+class FakeAdapter
+  def self.define_column_methods(*args)
+    # no-op
+  end
+
+  include ActiveRecord::ConnectionAdapters::MySQL::ColumnMethods
+end
+
+module ActiveRecord
+  module ConnectionAdapters
+    module MySQL
+      RSpec.describe ColumnMethods do
```

Comment:
why the weird nesting?

Reply:
It's not weird. It's more robust than `RSpec.describe ActiveRecord::ConnectionAdapters::MySQL::ColumnMethods` against the possibility that the autoloader doesn't like something.